examples/prometheus/prometheus.yaml

```diff
@@ -409,6 +409,22 @@ objects:
           action: replace
           target_label: kubernetes_name
 
+      - job_name: 'openshift-template-service-broker'
+
+        scheme: https
+        tls_config:
+          # TODO: specify CA
```

Comment:
@deads2k I think the TSB serving cert is signed by the service-signer CA - is it possible to get hold of that CA cert in the context of the prometheus pod?

Reply:
> @deads2k I think the TSB serving cert is signed by the service-signer CA - is it possible to get hold of that CA cert in the context of the prometheus pod?

It's injected into the SA token secret.  It's the CA that isn't ca.crt.